### PR TITLE
Commerce 7414 order rule validator

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-rule-service/src/main/java/com/liferay/commerce/order/rule/service/impl/CommerceOrderRuleEntryServiceImpl.java [hunk 1]
```diff
@@ -127,17 +137,26 @@ public CommerceOrderRuleEntry deleteCommerceOrderRuleEntry(
 
 	@Override
 	public CommerceOrderRuleEntry updateCommerceOrderRuleEntry(
-			long commerceOrderRuleEntryId, boolean active, String description,
-			String name, int priority, String typeSettings)
+			long userId, long commerceOrderRuleEntryId, boolean active,
```
Comment (riccardo-alberti) [inline modules/apps/commerce/commerce-order-rule-service/src/main/java/com/liferay/commerce/order/rule/service/impl/CommerceOrderRuleEntryServiceImpl.java:141]: @Commando18 didn't we remove the userId from the remote service?

**Review by riccardo-alberti — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Pull request closed without merging**